crates/rerun/src/run.rs

```diff
@@ -539,3 +548,35 @@ pub fn setup_ctrl_c_handler() -> (tokio::sync::broadcast::Receiver<()>, Arc<Atom
     .expect("Error setting Ctrl-C handler");
     (receiver, shutdown_return)
 }
+
+// ----------------------------------------------------------------------------
+
+use re_log::external::log;
+
+struct StrictLogger {}
```

Comment:
nit: this file here is large enough, `StrictLogger` could have gone to a different module